pwiz_tools/Skyline/Model/Import.cs

```diff
@@ -3463,9 +3463,14 @@ public static IEnumerable<FastaData> ParseFastaFile(TextReader reader, bool read
             string line;
             string name = string.Empty;
             StringBuilder sequence = new StringBuilder();
+            int lineNum = 0;
 
             while ((line = reader.ReadLine()) != null)
             {
+                for (int i=0; i < line.Length; ++i)
+                    if (line[i] < 32 || line[i] > 126)

```

Comment:
You need to allow character number 10 here. The FASTA spec says that description lines should contain character number 10 to separate the protein name from the description.

Reply:
You mean linefeed/newline? I don't think we would ever get that here because we're using ReadLine, right? Did you mean character 9, horizontal tab? Because that one is quite probable so it needs to be added.

Reply:
I think maybe the character I was thinking of is actually ASCII code 1, not 10.

If you look at "WebEnabledFastaImporter.cs", there's this code:
`            String[] alternatives = line.Substring(1).Split((char) 1);`
Meanwhile, the [Wikipedia page](https://en.wikipedia.org/wiki/FASTA_format) says:

> In a deprecated practice, the header line sometimes contained more than one header, separated by a ^A (Control-A) character.

I think the Wikipedia page confused me, and the special character I was thinking of was 1 not 10.

Reply:
Yeah, I just had to read up on the caret notation. I thought it was a Windows thing but it's older than that. I have never seen a FASTA with ^A in it, have you? The original spec also says the header line (really a comment) could start with `;` instead of '>' and there can be any number of comments before and between sequence lines. But I think very few modern parsers would support that. :/

Reply:
Updated code supports ^A and adds a test with it.